src/UnitTests/ArraysAndLists.cs

```diff
@@ -673,71 +674,6 @@ public void Should_assign_the_value_directly()
             _source.Values.ShouldBe(_destination.Values);
         }
     }
-
-    public class When_mapping_to_a_custom_collection_with_the_same_type_not_implementing_IList : AutoMapperSpecBase
-    {
-        private Source _source;
-
-        private Destination _destination;
-
-        public class ValueCollection : IEnumerable<int>
-        {
-            private List<int> implementation = new List<int>();
-
-            public ValueCollection(IEnumerable<int> items)
-            {
-                implementation = items.ToList();
-            }
-
-            public IEnumerator<int> GetEnumerator()
-            {
-                return implementation.GetEnumerator();
-            }
-
-            IEnumerator IEnumerable.GetEnumerator()
-            {
-                return ((IEnumerable)implementation).GetEnumerator();
-            }
-        }
-
-        public class Source
-        {
-            public ValueCollection Values { get; set; }
-        }
-
-        public class Destination
-        {
-            public ValueCollection Values { get; set; }
-        }
-
-        protected override MapperConfiguration Configuration { get; } = new MapperConfiguration(cfg =>
-        {
-            cfg.CreateMap<Source, Destination>();
-        });
-
-        protected override void Establish_context()
-        {
-            _source = new Source { Values = new ValueCollection(new[] { 1, 2, 3, 4 }) };
-        }
-
-        protected override void Because_of()
-        {
-            _destination = Mapper.Map<Source, Destination>(_source);
-        }
-
-        [Fact]
-        public void Should_map_the_list_of_source_items()
-        {
-            // here not the EnumerableMapper is used, but just the AssignableMapper!

```

Comment:
If it's `IEnumerable` (except `string`), it's a collection, and it should be mapped as such.